lib/uo_cli.rb

```diff
@@ -327,6 +333,11 @@ def self.create_project_folder(dir_name, empty_folder = false, overwrite_project
       example_gem_download = open(example_gem_file, ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE)
       IO.copy_stream(example_gem_download, File.join(dir_name, gem_name))
 
+      # Download visualization file to user's local machine
+      vis_file_name = File.basename(visualization_file)
+      vis_download = open(visualization_file, ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE)
+      IO.copy_stream(vis_download, File.join(vis_dir_abs_path))
```

Comment:
@vtnate I have added the commands to download it to the local folder
All the tests that are failing should clear once the file is in added to example_files in master I am guessing

Reply:
let's push the example files to master and rerun the tests to confirm.

Reply:
I pushed the file to master example files, the visualization tests are passing locally but are failing in the CI.

Reply:
@kflemin @vtnate , the tests are failing because the ResultVisualization class in scenario gem which is being called for runnign the visualization tests is on the develop branch not in the latest release of the scenario gem the gemfile is pointing towards.

This should be good to merge once everything else looks okay.